src/utils/list.js

```diff
@@ -1,4 +1,12 @@
-export const range = (count, start = 0) => count <= 0 ? [] : Array(count).fill().map((_, i) => i + start);
+export const range = (start, end) => {
```

Comment:
It definitely works as-is, and is more similar to lodash's `range`, but if we wanted to mimic it exactly, `end` would be up to but not including.